### ISSUE TITLE
Is multiple topics in BCF markup supported?

### ISSUE THREAD

**Issue by sweco-sevsma** [buildingSMART/BCF-XML#359]
Hello,

I am using the BCF file standard to compile some information about bim objects. However, when I am creating the bcf-file I want to create several topics for the same object. Is this possible and will be supported by the standard and other programs implementing it?

And if the answer is yes, should I wrap it in a `<Topics>` element?

I was reading here: https://github.com/buildingSMART/BCF-XML/tree/release_3_0/Documentation#topic
And looking throught the examples in this repository. However, I could not find any information about this being supported or not.

Thanks in advance for any replies
// Markus

**Comment by GeorgDangl** [comment]
Hey @sweco-sevsma, multiple topics in a BCF file are supported. They all go into different folders inside the zip archive:
https://github.com/buildingSMART/BCF-XML/tree/release_3_0/Test%20Cases/v3.0/Markup/Related%20topics%20with%20both%20topics%20in%20the%20same%20file/unzipped

Each topic is independent of the others, so you're free to add whatever topics you like😀

**Comment by sweco-sevsma** [comment]
First off, thanks for the super quick reply!

Yeah I noticed that and some other things right after I created the ticket. I should've closed it then but the issue for me is that when I add multiple topics with error information about the same object many times the file size become too large. But I solved it by using comments instead since they all refer to the same viewport and object.

Thank you anyway for the great documentation here on Github!
